From: Guy Hughes <g@gxg.me>
Date: Thu, 10 Dec 2015 23:53:41 -0500
Subject: Drop Bash Completion Shebang

---
 completion/tmuxinator.bash | 2 --
 1 file changed, 2 deletions(-)

diff --git a/completion/tmuxinator.bash b/completion/tmuxinator.bash
index 05ba7bb..fe2ead2 100755
--- a/completion/tmuxinator.bash
+++ b/completion/tmuxinator.bash
@@ -1,5 +1,3 @@
-#!/usr/bin/env bash
-
 _tmuxinator() {
     COMPREPLY=()
     local word
